### ISSUE TITLE
Increase coverage of pyvw.py

### ISSUE THREAD

**Issue by Smit-create** [VowpalWabbit/vowpal_wabbit#2268]
#### Describe the bug
The code coverage report of `pyvw.py` is:
```
----------- coverage: platform linux, python 3.7.4-final-0 -----------
Name                                                                                                           Stmts   Miss  Cover
----------------------------------------------------------------------------------------------------------------------------------
/home/smit/.local/lib/python3.7/site-packages/vowpalwabbit-8.8.0-py3.7-linux-x86_64.egg/vowpalwabbit/pyvw.py     417    227    46%

```
An html code coverage report can be generated locally with:
`/vowpal_wabbit/python$ pytest --cov=vowpalwabbit.pyvw --cov-report=html tests/test_pyvw.py`

Below is the code coverage report of `sklearn_vw.py` which looks good:
```
----------- coverage: platform linux, python 3.7.4-final-0 -----------
Name                                                                                                                 Stmts   Miss  Cover
----------------------------------------------------------------------------------------------------------------------------------------
/home/smit/.local/lib/python3.7/site-packages/vowpalwabbit-8.8.0-py3.7-linux-x86_64.egg/vowpalwabbit/sklearn_vw.py     163     20    88%

```
Does increasing the code coverage report seems to be a good idea to work upon?

**Comment by jackgerrits** [comment]
Adding tests is always welcome! 😄

**Comment by Smit-create** [comment]
Sure! will open a PR for that to work upon.

**Comment by Smit-create** [comment]
I have started working on it. But without documentation is hard to run them. In `SearchTask()`, that type of object should be `sch`?

**Comment by jackgerrits** [comment]
The search objects are complex, I would suggest you skip over them for now. If you would like to learn more have a look at these resources:
- https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Learning-to-Search-Sub-System
- https://hunch.net/~l2s/

**Comment by Smit-create** [comment]
Okay!, will look over that

**Comment by jackgerrits** [comment]
Looks like this was addressed in #2276
